src/Views/Lobby.hs

```diff
@@ -163,7 +138,7 @@ addGameToDOM api gameName = do
   appendChild documentBody gameDiv
 
   clickEventString gameName $ do
-    bool <- onServer $ joinGame api <.> gameName
+    bool <- onServer $ joinGame api <.> gameName <.> ""
```

Comment:
`bool` is never used. You don't need to to receive a value just because the function returns something.